src/main/java/io/jenkins/plugins/orka/helpers/OrkaClientProxy.java

```diff
@@ -67,21 +67,31 @@ public OrkaClientProxy(String endpoint, String credentialsId, int httpClientTime
 
     public ConfigurationResponse createConfiguration(String vmName, String image, String baseImage,
             String configTemplate, int cpuCount) throws IOException {
+        return this.createConfiguration(vmName, image, baseImage, configTemplate, cpuCount, null);
+    }
+
+    public ConfigurationResponse createConfiguration(String vmName, String image, String baseImage,
+            String configTemplate, int cpuCount, String scheduler) throws IOException {
 
         try (OrkaClient client = getOrkaClient()) {
-            return client.createConfiguration(vmName, image, baseImage, configTemplate, cpuCount);
+            return client.createConfiguration(vmName, image, baseImage, configTemplate, cpuCount, scheduler);
         }
     }
 
     public DeploymentResponse deployVM(String vmName) throws IOException {
-        try (OrkaClient client = getOrkaClient()) {
-            return client.deployVM(vmName);
-        }
+        return this.deployVM(vmName, null, null);
```

Comment:
`nit`: You can call the second overload (`this.deployVM(vmName, null);`) It will make sure to call the third.
This is how usually overloads call each other - from the less specific to the more specific one.
And this way you do not need to change all overloads if you are to introduce a new one. You only change the last one.